### ISSUE TITLE
v1.1.0 Connection Refused at Step 6 (Installation in NAND Flash)

### ISSUE THREAD

**Issue by naigel89** [0x1iii1ii/PPPwn-Luckfox#28]
Dispite been following the last 5 steps, I recieve an error when I tried to connect to the device using MobaXTerm. The configuration of the device its exactly as it's written at the "readme". Anyone found a solution?? 

I'll be reading.

![image](https://github.com/0x1iii1ii/PPPwn-Luckfox/assets/135312384/886c6b84-c0e5-40b1-afc2-5d244751eded)

**Comment by 0x1iii1ii** [comment]
try to disable your windows firewall and try again.

**Comment by naigel89** [comment]
Same problem again

**Comment by 0x1iii1ii** [comment]
try to burn buildroot image again, mentioned in step 2.

**Comment by Regis242** [comment]
@0x1iii1ii the luckfox connected to the PC, we should also connect the network cable?

**Comment by Paskale84** [comment]
same issue, reflash nand and resolve. only usb__

**Comment by naigel89** [comment]
Yes I know. The problem was that I skipped that step. I have a Luckfox Pico Plus and, following the instructions, it's written that you don't need to do the second step. 

First step: "If you got Luckfox pico Plus/Pro/Max, you don't need to burn the OS, Luckfox already has pre-installed buildroot right out of the box, so let's move to step 3.". 

So well, apparently, it is. After doing it, the issue was solved. Thank you.
